run_gunicorn.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
-echo "Launching gunicorn application..."
-poetry run flask run --host=0.0.0.0
-#poetry run gunicorn --bind 0.0.0.0:5000 wsgi:app
+echo "Launching application from Gunicorn..."
+poetry run gunicorn --bind 0.0.0.0:5000 todo_app.app:create_app\(\)
```

Comment:
Personal preference rather than important, but we could remove the escape characters by moving the module reference inside quotes:
```bash
poetry run gunicorn --bind 0.0.0.0:5000 "todo_app.app:create_app()"
```

Reply:
that's MUCH BETTER, thanks!